packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -192,14 +218,17 @@ $btn-disabled-bgColor                  : $color-disabled-01;
 $btn-disabled-textColor                : $color-content-disabled;
 
 $btn-sizeLarge-font-size               : 'heading-s';
-$btn-sizeLarge-padding                 : 13px 1.2em 15px;
+$btn-sizeLarge-padding                 : 14px 24px;
 $btn-sizeLarge-loading-color           : $color-content-interactive-primary;
 $btn-sizeLarge-loading-colorOpaque     : rgba($btn-sizeLarge-loading-color, $btn-default-loading-opacity);
+$btn-sizeLarge-iconHeight              : 23.5px;
```

Comment:
Might be worth checking with design comparing with a real icon design, but it looks like the icon size itself in the button designs for large icons is `18 x 18` which is inside a `24 x 24` bounding box (so padded out by 6px).

Medium is then 18x18 (with 24x24 bounding box), small is `15x15` (bounding box is 20px) and small is 12x12 (bounding box of 16px).